assets/vue/components/ResultSummary.vue

```diff
@@ -32,6 +32,7 @@ export default {
           searchParams.append('flavor', flavor);
         });
       }
+      searchParams.delete('distri');
```

Comment:
do you know where the distri key comes from? It should be possible to specify multiple query keys, e.g. `distri=sle&distri=opensuse`. Maybe it's better to handle this at the source of the problem, not in the UI layer.

Reply:
It comes from the dashboard database as `linkinfo` data structure via JSON API. So this might actually be an ok solution.

Reply:
[This](https://github.com/openSUSE/qem-dashboard/blob/08cea810f936faeb6af35b645270d85f6569c6b9/t/json.t#L122) is what it looks like in tests. Hiding information in the API will probably hurt us later when we need it for another feature.

Reply:
[This](https://github.com/openSUSE/qem-dashboard/blob/08cea810f936faeb6af35b645270d85f6569c6b9/t/json.t#L122) is what it looks like in tests. Hiding information in the API will probably hurt us later when we need it for another feature.

Reply:
Yes, that is definitely possible (e.g. https://openqa.suse.de/tests/overview?build=%3A30272%3Agcr&distri=sle&distri=opensuse&groupid=432) and it would also make much more sense.

Not sure why the distri is wrong sometimes in the first place (e.g. the link mentioned before had `distri=sle` in the dashboard) but we should likely tackle that problem on the source. Maybe this is due to some kind of grouping/folding that is happening with the data and we need to make sure to preserve all different distributions during the process.

Of course **if** the distri is completely irrelevant for how the entries on the blocked page are displayed then just removing the parameter would make kind of sense (because then we probably really just want to link to an overview where the distri doesn't matter as well). Even then it would probably make sense to ensure `this.result.linkinfo` does not have the distri in the first place (if that's possible, not sure where that value comes from).

Reply:
I haven't read @kraih 's comment before writing mine. Maybe the deletion on frontend-level is the way to go then. It would still make sense for me to clear up the if mentioned in the last paragraph of my previous comment.

Reply:
The `distri` value should come directly from the job settings stored in the database. Which are submitted by the bot to the dashboard.

Reply:
@michaelgrifalconi what about "It would still make sense for me to clear up the if mentioned in the last paragraph of my previous comment."

Reply:
I guess I can have a look at the code myself.

Reply:
Just judging by the code it looks like the blocked page is completely agnostic to the distribution. So I suppose it would never filter or group by the distribution. I guess then it makes also sense to just delete this parameter in the frontend. So I guess we can merge this (although it would be nice to extend the checks in the UI test).